src/vs/workbench/contrib/interactiveSession/browser/actions/interactiveSessionCodeblockActions.ts

```diff
@@ -118,12 +125,56 @@ export function registerInteractiveSessionCodeBlockActions() {
 				return;
 			}
 
-			const activeTextModel = textFileService.files.get(activeModel.uri);
+			// Check if model is editable, currently only support untitled and text file
+			const activeTextModel = textFileService.files.get(activeModel.uri) ?? textFileService.untitled.get(activeModel.uri);
```

Comment:
Re https://github.com/microsoft/vscode/pull/178632#discussion_r1153865674

we probably don't have a better way to check readonlyness as we always allow edits to `ITextModel` (which enables Live Share). Checking files and untitled is the best thing we can do here.